pkg/kubelet/kubeletconfig/configfiles/configfiles_test.go

```diff
@@ -178,6 +194,111 @@ foo: bar`),
 	}
 }
 
+func TestLoadInstance(t *testing.T) {
+	cases := []struct {
+		desc      string
+		file      *string
+		expect    *kubeletconfigv1beta1.KubeletInstanceConfiguration
+		err       string
+		strictErr bool
+	}{
+		//{
```

Comment:
instead of an error it returns an external object with, `typeMeta.Kind = ""`, need to check how to return a strict error here.